frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -1,10 +1,13 @@
 import React, { useEffect, useState } from "react";
 import { getSightings, Sighting } from "../../clients/apiClient";
 import { SightingCard } from "./SightingCard";
+// import {useNavigate} from "react-router-dom";
 import "./BrowseSightings.scss";
+import { MapPage } from "./MapPage";
 
 export const BrowseSightings: React.FC = () => {
   const [sightings, setSightings] = useState<Sighting[]>();
+  const [mapToggle, setMapToggle] = useState(false);
```

Comment:
"Toggle" is a verb, so this should be renamed to something like
```suggestion
  const [isShowingMap, setIsShowingMap] = useState(false);
```